app/js/components/LabResultsList.jsx

```diff
@@ -327,7 +327,12 @@ export class LabResultsList extends PureComponent {
     const getPatientLabResults = () => {
       // build a list of all obs from the encounters that are of type LabSet or Test
       return encounters.reduce((acc, encounter) => {
-        const obs = encounter.obs ? encounter.obs.filter(o => isLabSet(o) || isTest(o)) : [];
+        // flattens up to 3 levels of nested obs groups (TODO: better way to do this?)
```

Comment:
Maybe using recursion until o.groupMembers is null?

Reply:
Yeah, was thinking of recursion, was trying to keep it simple/easy and just use the flatmap stuff, but let me work on it.